relay/kusama/src/lib.rs

```diff
@@ -1217,6 +1224,8 @@ impl parachains_paras::Config for Runtime {
 	type QueueFootprinter = ParaInclusion;
 	type NextSessionRotation = Babe;
 	type OnNewHead = Registrar;
+	// TODO:(PR#159)(PR#1694): check `AssignCoretime` bellow and remove this comment!
```

Comment:
there are few new todos, are you doing to resolve them?

Reply:
those are the last open point from description and @antonva should be working on them see [comment](https://github.com/polkadot-fellows/runtimes/pull/159#issuecomment-1942479264)